webapp/src/pages/createForm/components/Section.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import Question from './Question';
+
+const Section = ({
+  t, num, setSection, sections, inputs,
+  addSection, addQuestion
+}) => {
+  const [input, setInput] = useState({
+    name: inputs.name,
+    description: inputs.description,
+    id: inputs.id,
+    questions: inputs.questions
+  });
+
+  const handleChange = (prop) => (e) => {
+    setInput({...input, [prop]: e.target.value});
+  }
+  const updateSections = () => { // Update sections in the application form when section loses focus
+    sections[num-1] = input;
+    setSection([...sections]);
+  }
+  const handleQuestions = (inpt) => {
+    setInput({...input, questions: inpt})
+  }
+  return (
+    <>
+      <div
+        className="section-wrapper"
+        // onBlur={updateSections}
+      >
+        <div className="section-number">
+          <label>{t('Section') + ` ${num} ` + t('of') + ` ${sections.length}`  }</label>
```

Comment:
FYI: No need to change this, but for future reference the [`Trans` component](https://react.i18next.com/#what-does-my-code-look-like) allows you to avoid breaking up the strings like this.